docs/manual/docs/cmd/spo/list/list-sitescript-get.md

```diff
@@ -0,0 +1,44 @@
+# spo list sitescript get
+
+Extracts a site script from the SharePoint list
+
+## Usage
+
+```sh
+spo list sitescript get [options]
+```
+
+## Options
+
+Option|Description
+------|-----------
+`--help`|output usage information
+`-u, --webUrl <webUrl>`|URL of the site where the list to extract the site script from is located
+`-l, --listId [listId]`|ID of the list to extract the site script from. Specify either `listId` or `listTitle` but not both
+`-t, --listTitle [listTitle]`|Title of the list to extract the site script from. Specify either `listId` or `listTitle` but not both
+`-o, --output [output]`|Output type. `json|text`. Default `text`
+`--verbose`|Runs command with verbose logging
+`--debug`|Runs command with debug logging
+
+!!! important
+    Before using this command, log in to a SharePoint Online site, using the [spo login](../login.md) command.
+
+## Remarks
+
+To extract a site script from a list, you have to first log in to a SharePoint Online site using the [spo login](../login.md) command, eg. `spo login https://contoso.sharepoint.com`.
+
+## Examples
+
+  Extract a Site Script from an existing SharePoint List with title _CustomList_
```

Comment:
List name in the description (CustomList) doesn't match the list name in the command (ContosoList)